src/ui/server/controllers/insurance/policy-and-export/map-and-save/index.ts

```diff
@@ -5,7 +5,7 @@ import { Application, RequestBody, ValidationErrors } from '../../../../../types
 
 /**
  * mapAndSave
- * Map and save any valid Single contract policy fields
+ * Map and save any valid  policy fields
  * @param {Express.Request.body} Express request body
  * @param {Object} Application
```

Comment:
same here

Reply:
Discussed - we have this in many places, will update all in one go.